### PR TITLE
The use of the modulefile has been deprecated, a metadata,json should be used instead.

### PR DESCRIPTION
**Pull request by attachmentgenie**
small change but will help us tremendously in our ci setup

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "richardc-datacat",
+  "version": "0.6.1",
+  "author": "richardc",
+  "summary": "Puppet type for handling data fragments",
+  "license": "Apache 2.0",
```
Comment (richardc) [inline metadata.json]: Can you make this `Apache-2.0` instead as metadata-json-lint complains.

```
$ metadata-json-lint metadata.json
Warning: License identifier Apache 2.0 is not in the SPDX list: http://spdx.org/licenses/
Errors found in metadata.json
```

**Comment by attachmentgenie** [comment]
just fixed the lint warning in 0234eea

**Comment by richardc** [comment]
Please squash these together.

**Comment by attachmentgenie** [comment]
I am to dumb to squash properly/correctly, easier to start again a create a new pull request

**Pull request closed without merging**